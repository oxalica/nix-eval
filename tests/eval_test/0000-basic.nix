{
  hello = "world";
  paths = [ <nixpkgs> /* ./nix not implemented */ /nix http://nix ];
  cond = if 0 == 0 then 42 else 4.2;
  foo = {}.a or { b = 42; }.b;
  buildPhase = ''
    echo "${"hello" + "world"}"
  '';
}
