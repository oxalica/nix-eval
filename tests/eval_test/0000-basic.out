{
  buildPhase = "echo helloworld";
  cond = 42;
  foo = 42;
  hello = "world";
  paths = [
    "/path/to/nixpkgs"
    "/nix"
    "http://nix"
  ];
}